skins/oasis/modules/templates/CommentsLikes_Index.php

```diff
@@ -1,29 +1,29 @@
 <? // render simple version for comments bubble (for blogs) ?>
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)